targettests/execution/qubit_management/if-1.cpp

```diff
@@ -9,10 +9,8 @@
 #include <cudaq.h>
```

Comment:
nit: can we remove this extra `qubit_management` directory here? rename the tests `qubit_management_if-0.cpp`, etc.

Reply:
Yes, but I'd prefer that Adam do that in his upcoming PR since he is planning on adding some of his tests anyway. (If I rename them out from underneath him, then that might complicate his merge.)